t3/schema.py

```diff
@@ -185,6 +185,14 @@ class RMGDatabase(BaseModel):
     class Config:
         extra = "forbid"
 
+    @validator('chemistry_sets')
```

Comment:
Instead of hard-coding the allowed chemistry_sets values in the validator, consider dynamically checking if the value of chemistry_sets is defined in libraries.yml.
`libraries_dict = read_yaml_file(path=os.path.join(DATA_BASE_PATH, 'libraries.yml'))
allowed_values = libraries_dict.keys()`